Add new sections to Methods and include commands for each tool. (#30)
EDUCE-UBC/MICB425: data.Rmd

-  `seq2lineage_Tables`

    This directory contains two files that will be used for treesapp update. One file contains the lineage information from the MAGs, and the other contains information for the SAGs. These tables were generated by taking each separate MAG and SAG respectively and passing them through the Genome Taxonomic Database Toolkit v1.4.0 classify workflow (see [Methods]). Then, the first two columns of the archaeal and bacterial summary files were taken and combined into each of these files.

## Methods

### Metagenomes and metatransriptopmes

#### Sample Collection

> Water was collected at 10, 100, 120, 135, 150, 165, and 200m at Saanich Inlet on August 1st, 2012 and filtered through a 0.22 µm Sterivex filter to collect biomass. Total genomic DNA and RNA were extracted from these filters. RNA was reversed transcribed into cDNA and both genomic DNA and cDNA were used in the construction of shotgun Illumina libraries. Sequencing data were generated on the Illumina HiSeq platform with **2x150bp** technology. For more in-depth sampling and sequencing methods, please see @Hawley.2017.

Author: Ok, I can change/update that.

```
trimmomatic -threads 8 -phred33 \
Path_to/SI072_XXXm/Raw/forward_dir/SI072_XXXm_R1.fastq \
Path_To/SI072_200m/Raw/reverse_dir/SI072_XXXm_R2.fastq \

Path_To/SI072_200m//trim/SI072_XXXm_pe.1.fq \
Path_To/SI072_200m//trim/SI072_XXXm_se.1.fq \
Path_To/SI072_200m//trim/SI072_XXXm_pe.2.fq \
Path_To/SI072_200m//trim/SI072_XXXm_se.2.fq \

ILLUMINACLIP:/usr/local/share/Trimmomatic-0.35/adapters/TruSeq3-PE.fa:2:3:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36
```

Trimmed and quality controlled reads were the then assembeld with SPAdes v3.9.0[ref] with standard parameters.

```
spades.py \
--sc \
-o Path_To_assembly_dir/ \
-1 Path_To_trim_dir/SI072_XXXm_pe.1.fq \
-2 Path_To_trim_dir/SI072_XXXm_pe.1.fq \
-s Path_To_trim_dir/$sample\_unpaired.fq \
-k $k_series \
--memory 90 \
--threads $threads \
--careful \
--cov-cutoff auto

```

#### SAG Assembly Quality Control and Decontamination

Quality of the SAGs was assessed using CheckM(v1.0.5) with the following command:

```
checkm lineage_wf --tab_table -x .fna --threads 8 --pplacer_threads 8
```

> If a SAG was >5% contamination, then the SAGs were passed through ProDeGe v2.3.0[ref].The resulting decontaminated SAG was re-assesed with CheckM. If the SAG still exceeded 5% contamination after several iterations, then short contigs <2000bp were removed from the assembly and re-assessed with CheckM. 

Reviewer: ```suggestion
SAGs found to have >5% contamination were passed through ProDeGe v2.3.0[ref].The resulting decontaminated SAGs were re-assesed with CheckM and those that still exceeded 5% contamination had all of their short contigs (i.e. <2000 bp) removed from the assembly. These were again re-assessed with CheckM to ensure they met our standards. 
```

> For this project, Based on the resulting CheckM scores, SAGs that were >50% Complete with <10% contamination were selected.

Reviewer: ```suggestion
For this project, SAGs that were estimated by CheckM to have >50% completeness and <10% contamination were selected.
```
